lib/jobs/ucs-service-profile-discovery.js

```diff
@@ -155,20 +155,20 @@ function UcsServiceProfileDiscoveryJobFactory(
 
         assert.string(data.name);
         assert.string(data.path);
-        var ids;
-        ids = [config.ucs, data.path];
+        var id = config.ucsHost + ':' + data.path;
+
         var obm = {
             config: config,
             service: 'ucs-obm-service'
         };
         var node = {
             type: type,
             name: data.name,
-            identifiers: ids,
+            identifiers: id,
```

Comment:
The `identifiers` is an array, not string (see https://github.com/RackHD/on-core/blob/master/lib/models/node.js#L34), please change it back.